vin/app/controllers/vin/monthly_selection_controller.rb

```diff
@@ -1,16 +0,0 @@
-class Vin::MonthlySelectionController < ApplicationController
-respond_to :json
```

Comment:
If the whole app is a json api, better put this in the `ApplicationController` and avoid repetition